src/PrestaShopBundle/Form/Admin/Sell/Address/ManufacturerAddressType.php

```diff
@@ -179,15 +192,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ]),
                     new Length([
                         'max' => AddressSettings::MAX_ADDRESS_LENGTH,
-                        'maxMessage' => $this->translator->trans(
+                        'maxMessage' => $this->trans(
                             'This field cannot be longer than %limit% characters',
-                            ['%limit%' => AddressSettings::MAX_ADDRESS_LENGTH],
-                            'Admin.Notifications.Error'
+                            'Admin.Notifications.Error',
+                            ['%limit%' => AddressSettings::MAX_ADDRESS_LENGTH]
                         ),
                     ]),
                 ],
             ])
             ->add('post_code', TextType::class, [
+                'label' => $this->trans('Zip/postal code', 'Admin.Global'),
```

Comment:
```suggestion
                'label' => $this->trans('Zip/Postal code', 'Admin.Global'),
```

Reply:
The two strings exist. June, the former Product Manager had decided to keep the version with the capital L, so I changed it to keep consistent in our wording decisions. ☺️